pontoon/administration/files.py

```diff
@@ -53,8 +66,8 @@ def split_comments_mine(
             cls.split_entities(
                 text, object, code=code, pointer=pointer, end=st0)
         groups = match.groups()
-        comment = silme.core.structure.Comment(
-            match.group(0)[1:].replace('\n#', '\n'))
+        comment = Comment()
+        comment.add(match.group(0)[1:].replace('\n#', '\n'))
```

Comment:
So after looking at silme's code and briefly at [this tutorial](https://wiki.mozilla.org/Silme:Tutorial:Concepts), this seems to be avoiding parsing entities out of comment nodes, so that the resulting L10nObject has just a single Comment node with the string contents inside of it. Is that correct?

If so, why bother removing this functionality?